api/v2/openapi.yaml

```diff
@@ -350,40 +348,75 @@ definitions:
       - name
       - value
       - isRegex
-  # Define required properties for 'alert'
-  alerts:
+  gettableAlerts:
     type: array
     items:
-      $ref: '#/definitions/alert'
-  alert:
-    type: object
-    properties:
-      startsAt:
-        type: string
-        format: date-time
-      updatedAt:
-        type: string
-        format: date-time
-      endsAt:
-        type: string
-        format: date-time
-      generatorURL:
-        type: string
-        format: uri
-      labels:
-        $ref: '#/definitions/labelSet'
-      annotations:
-        $ref: '#/definitions/labelSet'
-      receivers:
-        type: array
-        items:
-          $ref: '#/definitions/receiver'
-      fingerprint:
-        type: string
-      status:
-        $ref: '#/definitions/alertStatus'
-    required:
-      - labels
+      $ref: '#/definitions/gettableAlert'
+  # Shared properties in gettableAlert and postableAlert could also be
+  # de-duplicated via the `allOf` parameter like it is done with silences. So far
+  # this is blocked as the elm generator does not support it. Fore more details
+  # see https://github.com/OpenAPITools/openapi-generator/issues/1465.
+  gettableAlert:
```

Comment:
> @mxinden sounds good to me! As for naming, I prefer a simple Alert for the gettable alert and something different for creating an alert, maybe AlertCreate.

@w0rm I would prefer to be consistent with the silence naming scheme and not discriminate `postableAlerts` by being the only one being prefixed. What are your thoughts?

Reply:
@mxinden fine with me!